apps/theming/lib/Themes/DefaultTheme.php

```diff
@@ -120,12 +107,23 @@ public function getCSSVariables(): array {
 		$colorSuccess = '#2d7b41';
 		$colorInfo = '#0071ad';
 
+		// Chromium based browsers currently (2024) have huge performance issues with blur filters
+		$isChromium = $this->request !== null && $this->request->isUserAgent([Request::USER_AGENT_CHROME, Request::USER_AGENT_MS_EDGE]);
```

Comment:
Maybe also add mobile variant?

```suggestion
		$isChromium = $this->request !== null && $this->request->isUserAgent([Request::USER_AGENT_CHROME, Request::USER_AGENT_MS_EDGE, Request::USER_AGENT_ANDROID_MOBILE_CHROME]);
```

Reply:
Just tested and on android it does not suffer performance problems

Reply:
Android should have proper hardware acceleration, is is mostly a chromium + GPU driver issue.